viadot/sources/redshift_spectrum.py

```diff
@@ -79,8 +87,8 @@ def exists(self, database: str, table: str) -> bool:
         Check if a table exists in Glue database.
 
         Args:
-            database (str): AWS Glue catalog database name.
-            table (str): AWS Glue catalog table name.
+            database (str): Amazon Glue catalog database name.
+            table (str): Amazon Glue catalog table name.
```

Comment:
It's actually called AWS Glue, remember that Amazon is an e-commerce company, AWS is its cloud division. I think Redshift is one of the exceptions when they use the Amazon name, probably because it's quite old?

Reply:
Agreed. I've reverted previous commit and changes AWS S3 > Amazon S3 as per [📝 Minor changes](https://github.com/dyvenia/viadot/pull/636/commits/f8c1fcefb096de9daeb1785594afb86cd9a111d0)